docs/modules/spam_checker_callbacks.md

```diff
@@ -225,8 +239,9 @@ async def check_media_file_for_spam(
 ) -> bool
 ```
 
-Called when storing a local or remote file. The module must return a boolean indicating
-whether the given file can be stored in the homeserver's media store.
+Called when storing a local or remote file. The module must return a `bool` indicating
+whether the given file can be stored in the homeserver's media store. Return `False` if 
+to prevent this file from being stored; otherwise return `True`.
```

Comment:
I believe this one is incorrect, see https://github.com/matrix-org/synapse/blob/10a88ba91cb16ccf757984f0a7d41ddf8b4dc07f/synapse/rest/media/v1/media_storage.py#L148-L156

```suggestion
Called when storing a local or remote file. The module must return a `bool` indicating
whether the given file can be stored in the homeserver's media store. Return `True`
to prevent this file from being stored; otherwise return `False`.
```

Reply:
Thanks for spotting this. I propose wording this slightly differently---commit to follow shortly.